app/views/curation_concerns/file_sets/media_display/_universal_viewer.html.erb

```diff
@@ -0,0 +1,16 @@
+  <% if file_set.mime_type == "image/gif" %>
+    <%= image_tag thumbnail_url(file_set),
+          class: "representative-media",
+          alt: "",
+          role: "presentation" %>
+    <%= link_to main_app.download_path(file_set),
```

Comment:
@scherztc This will show the download button for a .gif no matter how large it is.  We need to include the check for file_is_too_large_to_download here.  So make this `link_to` look like https://github.com/uclibs/scholar_uc/blob/develop/app/views/curation_concerns/file_sets/media_display/_image.html.erb#L10-L18